shell/app/node_main.cc

```diff
@@ -169,7 +169,11 @@ int NodeMain(int argc, char* argv[]) {
     uv_loop_configure(loop, UV_METRICS_IDLE_TIME);
 
     // Initialize gin::IsolateHolder.
-    JavascriptEnvironment gin_env(loop);
+    bool setup_wasm_streaming =
+        node::per_process::cli_options->get_per_isolate_options()
+            ->get_per_env_options()
+            ->experimental_fetch;
```

Comment:
hm, it seems a little surprising to condition the WASM streaming stuff on experimental_fetch. is that the same condition as node.js?

Reply:
Visitor from the future reporting in: this copies upstream behavior @ https://github.com/nodejs/node/pull/42701